### DISCUSSION TITLE
Caddy reverse proxy with subpath

### DISCUSSION THREAD

**Discussion by Klaaktu** [dani-garcia/vaultwarden#3677] [Q&A] [answered]
```
  # Proxy everything to Rocket
  # if located at a sub-path the reverse_proxy line will look like:
  #   reverse_proxy /subpath/* <SERVER>:80
```
The wiki example says above, however I get "Page not found" when I try to visit the sub-path.
I think the sub-path might need to be explicitly stripped in Caddy ([see "Strip a path prefix" example](https://caddyserver.com/docs/caddyfile/directives/reverse_proxy#examples)). Will try later.
Also, I see the hyperlinks in the page does not include the sub-path, is there a way to include them?

**Comment by BlackDex** [comment, marked as answer]
You probably didn't configured the `DOMAIN` correctly.
It needs the sub-path too.

**Reply by Klaaktu** [reply]
Thanks, that was the cause.
Haven't read that far into the wiki but it's there: https://github.com/dani-garcia/vaultwarden/wiki/Using-an-alternate-base-dir

**Reply by iSanae1** [reply]
Thanks for your reply, this answer also helped me.
But I was a little confused about the `DOMAIN` in configure file.
- [Domain](https://en.wikipedia.org/wiki/Domain_name) should be look like  `example.org` not include the protocol and the path.
- `https://vaultwarden.example.com/base-dir` is more likely a URL, maybe `DOMAIN` should be renamed as `URL`.
